components/arm/server.go

```diff
@@ -16,6 +17,8 @@ import (
 	"go.viam.com/rdk/spatialmath"
 )
 
+var errUnimplemented = errors.New("unimplemented")
```

Comment:
`grpc.UnimplementedError` is used elsewhere in the codebase, would this work here?

Reply:
Actually thinking about this more its going to have to do more with the SDK error string than the GRPC one (I think).  Maybe this is fine for Go I'd have to check on what the the blessed error to return in this case is, but I'm also wondering what the convention for other SDKs is

Reply:
> grpc.UnimplementedError is used elsewhere in the codebase, would this work here?

If the geometries method returns `errors.New("unimplemented")` and this is how we check the error in server.go
```
if errors.Is(err, grpc.UnimplementedError) {
```
we **_will not_** make it into the body of the `if` statement.


----------------------------------------

If the geometries method returns `grpc.UnimplementedError` and this is how we check the error in server.go
```
if errors.Is(err, grpc.UnimplementedError) {
```
we **_will_** make it into the body of the `if` statement.


----------------------------------------


I think it makes most sense to keep doing
```
if errors.Is(err, errUnimplemented) {
```
where `errUnimplemented` = `errors.New("unimplemented")` since I do not expect people to be aware of `grpc.UnimplementedError`